src/mx/NotificationSenderInterface.java

```diff
@@ -0,0 +1,9 @@
+package mx;
+
+/**
+ * Created by masterix on 28.05.2017.
+ */
+public interface NotificationSenderInterface {
+    void setDomain(Domain domain);
+    void sendNotification();
```

Comment:
This method should accept a parameter - a `String` or binary blob, that would be sent. Details about how and where it is actually sent should be implementation detail in concrete classes implementing this interface